frontend/locales/ar/nav.json

```diff
@@ -0,0 +1,64 @@
+{
+  "Monitoring": "عربي Monitoring",
+  "Alerting": "عربي Alerting",
+  "Metrics": "عربي Metrics",
+  "Dashboards": "عربي Dashboards",
+  "Logging": "عربي Logging",
+  "Home": "عربي Home",
+  "Overview": "عربي Overview",
+  "Projects": "عربي Projects",
+  "Search": "عربي Search",
+  "Explore": "عربي Explore",
+  "Events": "عربي Events",
+  "Operators": "عربي Operators",
+  "Workloads": "عربي Workloads",
+  "Pods": "عربي Pods",
```

Comment:
Many of these items we probably don't want to translate as they're proper names. I think anything that is a k8s resource name probably should not be translated.

Reply:
The Kubernetes documentation ([Pods example](https://kubernetes.io/docs/concepts/workloads/pods/)) is actually localized, and I noticed things like pluralization differences in Chinese/Japanese compared to English/Spanish even though they're using English for the term. I'm wondering if it's still something that needs to be localized even if it's not translated? I don't know any Chinese and my Japanese is pretty bad. Do you think it would be useful to get translation to weigh in?